app/products/calls/actions/calls.ts

```diff
@@ -270,6 +282,17 @@ export const leaveCall = () => {
     setSpeakerphoneOn(false);
 };
 
+export const leaveCallPopCallScreen = async () => {
+    leaveCall();
+
+    // Need to pop the call screen, if it's somewhere in the stack.
+    await dismissAllModals();
```

Comment:
I wonder what modals may be visible in this situation, and why we have to pop down to the call.

The kind of modals (or screens) I am thinking about are stuff like a interactive dialog, or something like that, but maybe it is not applicable here.

Reply:
I was following what the other pop screen functions do in the code, and it didn't seem like it would hurt anything to be extra careful. At this point I'd prefer to keep it, just to be safe and not break anything unintentionally (since we don't have e2e tests atm).

Reply:
What is the expected behavior here? Then we'll use the information to determine if this is needed or not, at the moment I do not see a need for it, but I may be missing some context

Reply:
Yah, good point. I tested it and it looks like there can't be a modal on top when `leaveCallPopCallScreen` is hit. But we could have call screen in the stack, so we do need to rest of this fn. 
So I removed the `dismissAllModals`.

Reply:
@enahum Is this good to go, or are you thinking of something else?